src/js/modules/browse.js

```diff
@@ -208,7 +264,7 @@ function initBiomeFilter($div) {
     });
 }
 
-initBiomeFilter($('section').find('.tableFilters'));
+initBiomeFilter($('section.table-container:not(#publications-section)').find('.tableFilters'));
```

Comment:
I have merged this PR but could we please avoid statemantes like that? Coudl we get clear filter on the publication table too?